src/resources/wait-time.ts

```diff
@@ -0,0 +1,68 @@
+import { AxiosInstance } from 'axios';
+
+import Conditional from './conditional';
+
+export interface WaitTimeParameters {
+  location_id?: string | number;
```

Comment:
Can we change this to simply `location` in order to stick to the pre-set convention